Modifications to figure in Appendix

pllim opened the pull request:
Partially address #27  (this PR does not change the caption)

The `from matplotlib ...` is more of a personal preference here. See https://stackoverflow.com/questions/11469336/what-is-the-difference-between-pylab-and-pyplot .

keflavich commented on main.tex:
```diff
@@ -323,7 +323,7 @@ \section{Example}
     from astroquery.skyview import SkyView
     from astroquery.vizier import Vizier
-    import pylab as pl
+    from matplotlib import pyplot as pl
```
:+1: to this

bsipocz replied:
Super nitpicking, but according to astropy coding guidelines, this should be `import matplotlib.pyplot as plt`

keflavich commented on main.tex:
```diff
@@ -341,7 +341,7 @@ \section{Example}
     ax = fig.add_axes([0.15, 0.1, 0.8, 0.8], projection=mywcs)
     ax.set_xlabel("RA")
-    ax.set_ylabel("Dec")
+    ax.set_ylabel("DEC")
```
:-1: to this.  RA is capitalized because it's an acronym.  Dec is not capitalized because it's an abbreviation.

keflavich replied:
(so could you revert this change, then I'll merge the PR?)

pllim commented:
Using GitHub's web interface to resolve conflict created a merge commit. I'll squash and re-push. Hold one.

pllim commented:
Okay. I have addressed all the comments. Since we want to be consistent with Astropy snippets, I also changed `coordinates` import to `as coords`. Thank you all for the reviews!

keflavich commented:
I'm OK with merge commits showing up in the history.

But, yes, this looks good to go.  I still stick to `import pylab as pl` (or `import matplotlib.pyplot as pl`), but I realize I'm now in the minority...
